content/docs/command-reference/params/index.md

```diff
@@ -97,14 +97,14 @@ process:
   bow: 15000
 ```
 
-Using `dvc run`, define a [stage](/doc/command-reference/run) that depends on
-params `lr`, `layers`, and `epochs` from the params file above. Full paths
+Using `dvc stage add`, define a [stage](/doc/command-reference/run) that depends
+on params `lr`, `layers`, and `epochs` from the params file above. Full paths
```

Comment:
Yeah actually, do we need an issue to review all the places we implicitly recommend `stage add` vs. manual editing of dvc.yaml? Should check in fact it may already exits...

Reply:
UPDATE on this: We do - https://github.com/iterative/dvc.org/issues/2076